ansible/setup/ansible.md

```diff
@@ -51,6 +50,18 @@ Each playbook uses one or more modules. Modules encapsulate operations that run
 them idempotent. There are a lot of existing modules, which you can read about at
 http://docs.ansible.com/modules_by_category.html.
 
+## Ansible Config Settings 
+
+The design of the DE's ansible variables makes use of YAML
+["hashes"](http://docs.ansible.com/ansible/YAMLSyntax.html#yaml-basics). If you ever intend to 
+override a single value in a hash, you will need to set ansible's hash behaviour to _"merge"_.
+The easiest way to set these are with environment variables.
+For example, add these lines to your `~/.bash_developer` or `~/.bashrc` file:
```

Comment:
`.bash_developer` is specific to our development environment setup. Maybe it would be better to mention `.profile` or `.bash_profile` in addition to `.bashrc` instead.